templates/admin-page.php

```diff
@@ -17,12 +17,12 @@
 
 			<form>
 				<h2>
-					<label class="title" for="plugin-check__plugins">
+					<label class="title" for="pc_plugins">
 						<?php esc_html_e( 'Check the Plugin', 'plugin-check' ); ?>
 					</label>
 				</h2>
 
-				<select id="plugin-check__plugins" name="plugin_check_plugins">
+				<select id="pc_plugins" name="plugin_check_plugins">
```

Comment:
Is there a reason why these IDs need to change?

Reply:
@joemcgill Not any specific reason, but it is changed for simplicity, as we have button ID `pc_check_it`, so same way, I have changed the ID of the select box, as the short IDs are good to handle in the JS file. And I made this template, and that ID is not yet used anywhere, so we can change it, as it will not break anything. 🙂

Reply:
Same question from my end, I think the original IDs here were more clear and had less of a chance for conflict. There's a higher chance that some other plugin uses the `pc_` prefix than another plugin using the `plugin-check_` prefix. Not a big deal, but I would advise to change it back. Maybe we should make it a bit more clear also what the element is for, e.g. `plugin-check__plugins-dropdown`.